tests/integration/features/chat/mentions.feature

```diff
@@ -623,3 +623,13 @@ Feature: chat/mentions
     And user "participant3" is participant of the following rooms (v4)
       | id         | unreadMention | unreadMentionDirect |
       | group room | 0             | 0                   |
+
+  Scenario: At-all in note-to-self broke the mention parsing
+    And user "participant1" creates note-to-self (v4)
+    And user "participant1" sends message "Test @all" to room "participant1-note-to-self" with 201
+    And user "participant1" is participant of the following rooms (v4)
+#      | id                        | type | name          |
+#      | participant1-note-to-self | 6    | Note to self  |
```

Comment:
type=6 is filtered out to not disturb the testing. I guess it would be cool to have an opt in on this

Reply:
Not sure I get this, why would an at-all be useful there?

Reply:
This comment is about "note to self" rooms being removed in the test framework.

The at-all on note-to-self broke, it throws a 500 on posting, on getting your rooms and getting the messages in the room 🙃
All fixed with this PR